bluesky/plans.py

```diff
@@ -1124,6 +1125,9 @@ def spiral_fermat(detectors, x_motor, y_motor, x_start, y_start, x_range,
         delta radius
     factor : float
         radius gets divided by this
+    dr_y : float, optional
+        Delta radius along the major axis of the ellipse, if not specifed
+        defaults to dr.
```

Comment:
I don't see it's assigned `dr` anywhere in the code below. Only `dr_y=dr_y` is used. Could you please point me to the place where this default is processed?

Reply:
It is done in a for loop in the `spiral_fermat` plan pattern [here](https://github.com/bluesky/bluesky/blob/99eb69cd2d55c42103607765c4439833fac8ee2e/bluesky/plan_patterns.py#L242) in a round about way. we actually use `dr_aspect` (`dr/dr_y`) to create the ellipticity in the plan pattern and set it to 1 (i.e `dr_y` = `dr` ) if `dr_y` is `None`.

Does this clarify your question @mrakitin?

Reply:
Thanks for the explanation, sounds good!